### PR TITLE
Accept debug events to log then

### PR DESCRIPTION
**Pull request by lbeschastny**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -37,9 +37,11 @@ pub fn init_tracing_with_fallbacks(
         .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
         .with_writer(std::io::stdout.with_max_level(log_level));
 
+    let level_filter =
+        LevelFilter::from_level(std::cmp::max(log_level, tracing::Level::INFO));
     let subscriber = tracing_subscriber::registry()
         .with(fmt_layer)
-        .with(LevelFilter::INFO)
+        .with(level_filter)
```
Comment (raphamorim) [inline src/lib.rs:44]: The thing is if isn't >=INFO the otel doesn't work.

Comment (lbeschastny) [reply src/lib.rs:44]: But that is what I'm doing with `max`

Comment (lbeschastny) [reply src/lib.rs:44]:
```
    let level_filter =
        LevelFilter::from_level(std::cmp::max(log_level, tracing::Level::INFO));
```

Comment (lbeschastny) [reply src/lib.rs:45]: Is it clearer now?

```rust
    let level_filter: LevelFilter = max(log_level, tracing::Level::INFO).into();
```

Comment (lbeschastny) [reply src/lib.rs:45]: Tested my changes locally with stdout and jaeger and everything seems to work fine

**Review by raphamorim — APPROVED** [review]

**Pull request merged**